dashboard/lib/metrics/events.rb

```diff
@@ -78,8 +78,8 @@ def log_event(user: nil, event_name:, event_value: nil, metadata: {}, get_enable
 
         if user.present?
           user_params[:user_id] = user.id.to_s
-          user_params[:custom_ids][:user_type] = user.user_type if user.user_type
-          user_params[:custom_ids][:enabled_experiments] = enabled_experiments if enabled_experiments
+          user_params[:custom][:user_type] = user.user_type if user.user_type
+          user_params[:custom][:enabled_experiments] = enabled_experiments if enabled_experiments
```

Comment:
Should not we use camel case for consistency with events on the JavaScript side, or does it not matter?

https://github.com/code-dot-org/code-dot-org/blob/bca9cb46f8151235f1ebb988d736dd1716d3af21/apps/src/metrics/StatsigReporter.js#L26

https://github.com/code-dot-org/code-dot-org/blob/bca9cb46f8151235f1ebb988d736dd1716d3af21/apps/src/metrics/StatsigReporter.js#L38

```suggestion
          user_params[:custom][:userType] = user.user_type if user.user_type
          user_params[:custom][:enabledExperiments] = enabled_experiments if enabled_experiments
```

Reply:
That's a good point. I'm not sure it completely matters, however Statsig's own docs on the [Statsig User](https://docs.statsig.com/server/rubySDK/#statsig-user) seem to suggest camelCase. It may be worth changing these (and potentially all the keys we set in the Statsig User).

Reply:
Statsig actually accepts either snake_case or camelCase keys for the Statsig User https://github.com/statsig-io/ruby-sdk/blob/179a171f6b6b97d58b634aec77a753c63a3c20be/lib/statsig_user.rb#L97-L107

Reply:
Yes, those are Statsig root attributes. I’m referring to the custom values we set. I don’t think they’ll be automatically formatted. Wouldn’t it be nice to format them to camelCase for consistency between backend and frontend events?

Reply:
Yep, I agree. I'll update the custom values to camelCase